integration-tests/jpa-postgresql-withxml/src/test/java/io/quarkus/it/jpa/postgresql/ImageMetricsITCase.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.it.jpa.postgresql;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.quarkus.test.junit.nativeimage.BuildOutputExtension;
+
+@ExtendWith(BuildOutputExtension.class)
+@QuarkusIntegrationTest
+public class ImageMetricsITCase {
```

Comment:
do I get it right that idea is to just add a test of these 17 characters in every IT test case and since the general structure is IT tests will include all the various extensions it will automagically works - no extra setup?

Reply:
Yes, if you follow the three steps mentioned in the description of this PR you should be good to go:

> 1. copying ImageMetricsITCase.java in the corresponding folder
> 2. running the test once
> 3. running the following script passing the build output json file as the first parameter.